react/hooks/useMouseSpeedDebouncer.ts

```diff
@@ -28,6 +30,8 @@ export const useMouseSpeedDebouncer = (
       const speed = getMouseSpeed()
       if (speed <= maxSpeed) {
         fn(...args)
+      } else if (lastSpeed !== null && lastSpeed >= minUnusualSpeed) {
```

Comment:
Why not use the maxSpeed also here?
```suggestion
      } else if (lastSpeed !== null && lastSpeed > maxSpeed) {
```

Reply:
Hi @danzanzini! 

You are right, it is a better approach. The change was made, thanks!